lib/hammer_cli_foreman/api/negotiate_auth.rb

```diff
@@ -0,0 +1,13 @@
+module HammerCLIForeman
+  module Api
+    class NegotiateAuth < ApipieBindings::Authenticators::Negotiate
+      def initialize(foreman_url, **options)
+        super("#{foreman_url}/users/extlogin", HammerCLI::SSLOptions.new.get_options(foreman_url).merge(options))
+      end
+
+      def user
```

Comment:
I'm getting at the end the message "Successfully logged in as 'UNKNOWN'." 
I wonder if it's the best message here or we could change  the method execute so that we see only "Successfully logged in"

Reply:
oh, I forgot to comment https://github.com/theforeman/hammer-cli-foreman/pull/555/files#diff-0d4650155f0335f18fca083b6c11c7a4c9a50d4af563f44d3d003caba468c030R39